index.html

```diff
@@ -2,9 +2,181 @@
 <html lang="en">
 <head>
 	<meta charset="UTF-8">
-	<title>Title</title>
+	<link rel="stylesheet" href="styles.css">
+	<title>Rekupe - Digital Agency in LA</title>
 </head>
 <body>
+<header class="header__outer-container">
+	<div class="header__inner-container">
+		<nav class="navigation">
+			<ul class="navigation__list">
+				<li class="navigation__list-items"><a class="navigation___link" href="#main__content-section">case studies</a></li>
+				<li class="navigation__list-items"><a class="navigation___link" href="#main__services--inner-container">services</a></li>
+				<li class="navigation__list-items"><a class="navigation___link" href="#footer__outer-container">contact us</a></li>
+			</ul>
+		</nav>
+		<div class="header__hero-section">
+			<h1>your website is an investment in the success of your business</h1>
+			<p class="header__hero-section-paragraph">We are here to help you succeed.</p>
+			<a href="#footer__outer-container">
+				<button class="main__cta-button">let's work together</button>
+			</a>
+		</div>
+		<div class="header__squiggle-line">
+			<img class="squiggle-line" src="/assets/img/squiggle_line_stripe.svg" alt="">
+		</div>
+	</div>
+</header>
+<div class="header--pseudo-skew"></div>
+<!--Start Main-->
+<main class="main__outer-container">
+	<div class="main__inner-container">
+		<section class="main__about-section">
+			<div class="main__paragraph--about-section">
+				<h2 class="main__paragraph--h2">
+					Rekupe is a digital agency in Los Angeles.
+				</h2>
+				<h2 class="main__paragraph--h2">
+					We use strategic UX, SEO optimization and beautiful visual
+					designs to help your business grow.
+				</h2>
+				<h2 class="main__paragraph--h2">
+					We see every project as a fresh opportunity, pushing ourselves to
+					create new digital experiences.
+				</h2>
+				<p class="main__paragraph--cta-line">
+					Have a question? <a href="#footer__outer-container">Click here</a> to contact us!
+				</p>
+			</div>
+		</section>
+		<!-- Articles -->
+		<div class="main__content-section--pseudo-skew"></div>
+		<section id="main__content-section">
+			<article class="main__content-article main__content-article--pseudo">
+				<section class="main__content-article--block">
+					<h2 class="main__content-article--header">rolling stone</h2>
+					<div class="main__content-article--pseudo-item-squiggle-line"></div>
+					<p class="main__content-article--paragraph">
+						You know Rolling Stone. It’s the definitive voice in music, politics and culture. It’s a global
+						icon.
+					</p>
+					<a class="content-article--link" href="projects.html">view project</a>
+				</section>
+				<section class="main__content-article--image content-article__image1">
```

Comment:
In plaats van een section tag kan je om de img een `<span>` tag gooien. Een span tag is een inline element wat er voor zorgt dat we meer vrijheid hebben met styling. Een section tag is net zoals een div een block element.